app/states/texas/validators/texas_address.py

```
@@ -140,3 +140,13 @@ def standardize_address(self):
                 self.standardized = _standardized
                 self.id = self.generate_key(self.standardized)
             return self
+
+
+class TECPersonAddressLinkModel(TECBaseModel, table=True):
+    __tablename__ = "tx_person_address_link"
+    __table_args__ = {"schema": "texas"}
```

Ruff (0.14.7)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)